README.md

```diff
@@ -136,6 +135,27 @@ Next, change default database connection to `"crate"`.
 'default' => 'crate',
 ```
 
+#### Configuration for HTTP Basic Auth
```

Comment:
@RatkoR These are the relevant changes to the README. Everything else is VSCode autoformatting.